### PR TITLE
[MIG][14.0] project_hr

### PR DESCRIPTION
**Pull request by AntoniRomera**
Standard migration

### CODE REVIEW

**Comment by AntoniRomera** [comment]
Can someone help me, on my local machine everything works correctly, but travis says that I can't search in a field "non stored field", I don't understand why in one place it works and not in another.

File: project_hr/models/project_task.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        "according categories at project level.",
+        store=False,
+    )
+    allowed_user_ids = fields.Many2many(
+        comodel_name="res.users",
+        string="Allowed users",
+        compute="_compute_allowed_user_ids",
+        help="Technical field for computing allowed users according employee "
+        "category.",
+        store=False,
```
Comment (ajaniszewska-dev) [inline project_hr/models/project_task.py:39]: problem with travis is here, you can set store to true or define search method

**Comment by ajaniszewska-dev** [comment]
use pre-commit to fix build

**Review by ajaniszewska-dev — APPROVED** [review]

File: project_hr/models/project_task.py [hunk 2]
```diff
@@ -28,15 +28,33 @@ class ProjectTask(models.Model):
...
+                    FROM res_users users
+                WHERE
+                    users.id = ANY(%s)
+            """,
+                (list(value),),
+            )
+
+        user_ids = self.env.cr.fetchone()[0] or []
+        return [("id", "in", user_ids)]
+
```
Comment (ernestotejeda) [inline project_hr/models/project_task.py:57]: Why do we need this method? allowed_user_ids a technical field intended to be used in a domain only.

File: project_hr/models/project_task.py [hunk 3]
```diff
@@ -28,15 +28,33 @@ class ProjectTask(models.Model):
         compute="_compute_allowed_hr_category_ids",
         help="Technical field for computing allowed employee categories "
         "according categories at project level.",
+        store=False,
```
Comment (ernestotejeda) [inline project_hr/models/project_task.py:31]: Computed fields are store=False by default, you don't need to add this line.

**Review by ernestotejeda — COMMENTED** [review]
Thanks for contributing @AntoniRomera.

- We need to have the pre-commit changes in a separated commit as expalned here https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#technical-method-to-migrate-a-module-from-130-to-140-branch

**Comment by ernestotejeda** [comment]
Hi @AntoniRomera , are you going to continue with this PR? If you can't, i can open a new PR to continue with this :) . Thanks.

**Comment by AntoniRomera** [comment]
> Hi @AntoniRomera , are you going to continue with this PR? If you can't, i can open a new PR to continue with this :) . Thanks.

All yours, I'm out of time

**Comment by pedrobaeza** [comment]
Superseded by #910

**Pull request closed without merging**